src/shared/components/stepper/StepperContext.tsx

```diff
@@ -1,4 +1,4 @@
-import { createContext, useCallback, useReducer } from "react";
+import { createContext, useCallback, useEffect, useReducer } from "react";
```

Comment:
the context now has a `data` object in state: `data: {0: {}, 1: {}...}`  which a step can lookup